scripts/jenkins/ardana/manual/deploy-crowbar.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+source lib.sh
+
+validate_input
+setup_ansible_venv
+mitogen_disable
+prepare_input_model
+prepare_infra
+
+trap exit_msg EXIT
+
+#build_test_packages
+bootstrap_crowbar
+# deploy_ses_vcloud
```

Comment:
What if SES is enabled ? this is also required.

Reply:
Right now I don t need to use SES when deploying so I didn't look around that. Can it stay as a follow-up task(new card in the backlog)?

Reply:
you should just call that function anyway. if you keep `ses_enabled` set to false, it won't do anything and you've nothing to worry about